client/src/main.js

```diff
@@ -21,10 +22,20 @@ if (!window.SANDTANK_SESSION_MANAGER_URL) {
 // ----------------------------------------------------------------------------
 
 Vue.config.productionTip = false;
-
 Vue.use(Vuex);
 Vue.use(Vuetify);
-
+Vue.config.productionTip = false;
+Vue.use(VueGtag,
+  {config: 
+     {
+       id: 'G-YY63PD0T3J',
+       params: {
+         send_page_view: false
```

Comment:
We may want this to be `true`. I know it's a SPA but it might be that setting this to false returns something weird when looking for page views. Probably not a big deal, I can check with the code we use to see how it handles it and if it's a problem we can update later.

Reply:
The initial suggestions I found for enable GA4 for Vue suggested setting send_page_view to false.
This is a blog discussion on the topic.
   https://developers.google.com/analytics/devguides/collection/ga4/views?client_type=gtag
Even after reading this I'm not quite sure whether I'm supposed to do this or not. Maybe it is safer remove this tag and let it do the default. I will remove this send_page_view: false tag.